src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyShareholderCountValidation.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.confirmationstatementapi.eligibility.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+import uk.gov.companieshouse.confirmationstatementapi.eligibility.EligibilityStatusCode;
+import uk.gov.companieshouse.confirmationstatementapi.eligibility.EligibilityRule;
+import uk.gov.companieshouse.confirmationstatementapi.exception.EligibilityException;
+import uk.gov.companieshouse.confirmationstatementapi.service.ShareholderService;
+
+public class CompanyShareholderCountValidation implements EligibilityRule<CompanyProfileApi> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CompanyShareholderCountValidation.class);
+
+    private final ShareholderService shareholderService;
+
+    public CompanyShareholderCountValidation(ShareholderService shareholderService) {
+        this.shareholderService = shareholderService;
+    }
+
+    @Override
+    public void validate(CompanyProfileApi companyProfile) throws EligibilityException {
+
```

Comment:
would you be able to add a feature flag similar to the Psc validation rule, that defaults to true. To allow us to bypass this eligibility checkif need be?